page-object.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |s|
   s.require_paths = ["lib"]
 
   s.add_dependency 'watir', '~> 6.8'
-  s.add_dependency 'selenium-webdriver', '~> 3.0'
+  s.add_dependency 'selenium-webdriver', '~> 3.142.3'
```

Comment:
@StephenTurley , is it necessary to change this runtime dependency? Isn't it only during development that we need 3.142.3?

Reply:
Hmm I thought there was a compatibility issue with ruby-2.5, but it seems fine locally. I'll change it back and see what CI says.

Reply:
Looks like `3.0` is fine in CI. Good catch!